backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/multitenancy/impl/NoMultiTenancyStrategyImpl.java

```diff
@@ -58,9 +58,9 @@ public String extractTenantScopedDocumentId(JsonObject hit) {
 	}
 
 	@Override
-	public void checkTenantId(BackendImplementor<?> backend, String tenantId) {
+	public void checkTenantId(String tenantId, FailureContext backendContext) {
 		if ( tenantId != null ) {
-			throw log.tenantIdProvidedButMultiTenancyDisabled( backend, tenantId );
+			throw log.tenantIdProvidedButMultiTenancyDisabled( tenantId, backendContext );
```

Comment:
Maybe `backendFailureContext`.

Reply:
Given the next commits that rename it to `EventContext`, not sure it's necessary? `EventContext backendContext` seems to make sense to me.